server/app/middlewares/orders.js

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
I am wondering why these are in a middleware folder? It seems a little redundant to re-write the same functions as before. Why not resolve the promises in the controller functions? I just think this might be a little to repetitive. What do you think?